SofaKernel/framework/sofa/core/visual/DrawToolGL.cpp

```diff
@@ -146,7 +146,7 @@ void DrawToolGL::drawLines(const std::vector<Vector3> &points, float size, const
 {
     if (points.size() != colors.size()*2)
     {
-        msg_warning("DrawToolGL") << "Sizes mismatch between points.size() and colors.size().";
+        msg_warning("DrawToolGL") << "Sizes mismatch, points.size(): " << points.size() << " should be equal to colors.size()*2: " << colors.size()*2;
```

Comment:
I dont think it is a good idea to output something in the drawtool, as the message will literally generate thousand (or more) lines in the console (or whatever)